npm/src/browser.ts

```diff
@@ -158,6 +158,17 @@ export function getLanguageServiceWorker(
   return createProxy(worker, wasmModule, languageServiceProtocol);
 }
 
+export { StepResultId, type IStructStepResult } from "../lib/web/qsc_wasm.js";
+export type {
+  IBreakpointSpan,
+  ICodeLens,
+  ILocation,
+  IOperationInfo,
+  IPosition,
+  IRange,
+  IStackFrame,
+  VSDiagnostic,
+} from "../lib/web/qsc_wasm.js";
```

Comment:
This isn't a new problem (I just spotted it in other code checked in over the past few months), but we used to always import from "../lib/node/qsc_wasm.js". The Node types are a (slightly) simpler subset of the web types which should work in all environments (but not vice-versa)

Don't worry about it here if a pain to change. I may go clean this up across the code-base later (and try to think how we could catch it)

Reply:
I don't mind taking care of it now, but, just to confirm:

We already have a _runtime_ import of `../lib/web/qsc_wasm.js` in this file. I believe that has to stay as `web`. Are you suggesting importing the types from `node`, but the runtime exports from `web`, even if they both occur in the same file?

If so, something like this 
```ts
import {
  IOperationInfo,
  TargetProfile,
  type VSDiagnostic,
} from "../../lib/web/qsc_wasm.js";
```

Has to turn into two separate import statements which seems a bit overkill.